### PR TITLE
Drools 3052 Implement read only mechanism for the cells in Scenario grid

### PR DESCRIPTION
**Pull request by gitgabrio**
@kkufova @Rikkola @danielezonca 
Scope of this PR is to implement read only mechanism for the cells in Scenario grid.
More specifically:
1) Index column should never be editable
2) description column should always be editable
3) GIVEN/EXPECTED columns should not be editable if not mapped to a real property
Different placeholder messages should appear in the different condition (empty editable cell vs not editable one)
Cell editing should start with single click

### CODE REVIEW

**Comment by gitgabrio** [comment]
@kkufova 
Should be fixed now

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/AbstractCommand.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                                                           ScenarioGridLayer gridLayer,
+                                                           String placeHolder) {
+        ScenarioHeaderTextBoxSingletonDOMElementFactory factoryHeader = FactoryProvider.getHeaderTextBoxFactory(scenarioGridPanel, gridLayer);
+        ScenarioSimulationBuilders.HeaderBuilder headerBuilder = getHeaderBuilder(title, columnId, columnGroup, factMappingType, factoryHeader);
+        return getScenarioGridColumn(headerBuilder, scenarioGridPanel, gridLayer, true, placeHolder);
+    }
+
+    protected ScenarioGridColumn getScenarioGridColumnLocal(ScenarioSimulationBuilders.HeaderBuilder headerBuilder) {
+        // indirection add for test
+        return getScenarioGridColumn(headerBuilder, scenarioGridPanel, scenarioGridLayer, false, ScenarioSimulationEditorConstants.INSTANCE.insertValue());
```
Comment (Rikkola) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/AbstractCommand.java]: Why do the tests need this?

Comment (gitgabrio) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/AbstractCommand.java]: @Rikkola 
static methods

**Comment by gitgabrio** [comment]
@manstis 
Hullo! Could merge this, squashing commits? Thx!!

**Pull request merged**